core/templates/domain/utilities/language-util.service.ts

```diff
@@ -149,25 +149,20 @@ export class LanguageUtilService {
     return language ? language.description : null;
   }
 
-  getContentLanguageDirection(languageCode: string): string {
-    let languageDirection = 'auto'; // Default auto for invalid languageCode
-    this.SUPPORTED_CONTENT_LANGUAGES.forEach((lang: ContentLanguage) => {
-      if (lang.code === languageCode) {
-        languageDirection = lang.direction;
+  getLanguageDirection(languageCode: string): string {
+    for (let i = 0; i < this.SUPPORTED_CONTENT_LANGUAGES.length; i++) {
+      const language = this.SUPPORTED_CONTENT_LANGUAGES[i];
+      if (language.code === languageCode) {
+        return language.direction;
       }
-    });
-    return languageDirection;
-  }
-
-  getAudioLanguageDirection(languageCode: string): string {
-    let languageDirection = 'auto'; // Default auto for invalid languageCode
-    this.SUPPORTED_AUDIO_LANGUAGES.forEach(
-      (audioLanguage: SupportedAudioLanguageBackendDict) => {
-        if (audioLanguage.id === languageCode) {
-          languageDirection = audioLanguage.direction;
-        }
-      });
-    return languageDirection;
+    }
+    for (let i = 0; i < this.SUPPORTED_AUDIO_LANGUAGES.length; i++) {
```

Comment:
Can you add a comment explaining why the first language direction match from SUPPORTED_CONTENT_LANGUAGES and SUPPORTED_AUDIO_LANGUAGES works here? It's non-obvious to someone who doesn't have full context (like me).

Reply:
I added a comment, see line 161 in `language-util-service.spec.ts`, let me know if it's descriptive enough.